src/AutoRest.CSharp/Common/Input/Inputs.cs

```diff
@@ -213,8 +213,9 @@ internal record InputListType(string Name, InputType ElementType, bool IsNullabl
 
     internal record InputDictionaryType(string Name, InputType KeyType, InputType ValueType, bool IsNullable = false) : InputType(Name, IsNullable) { }
 
-    internal record InputModelProperty(string Name, string? SerializedName, string Description, InputType Type, bool IsRequired, bool IsReadOnly, bool IsDiscriminator, FormattableString? DefaultValue = null, SerializationFormat SerializationFormat = SerializationFormat.Default)
+    internal record InputModelProperty(string Name, string? SerializedName, string Description, InputType Type, bool IsRequired, bool IsReadOnly, bool IsDiscriminator, SerializationFormat SerializationFormat = SerializationFormat.Default)
     {
+        public FormattableString? DefaultValue { get; init; }
```

Comment:
I moved this `DefaultValue` here because this property is never included in our cadl.json
Only the property bag related models will set this value, and those instances are created by calling the constructor in our generator.
To keep things clear, I moved this out of the constructor so that our cadl.json serializer does not have to deal with this.